app/main.py

```diff
@@ -80,15 +103,16 @@ async def swagger_ui_redirect():
     four_o_four.router,
     friendview.router,
     google_connect.router,
-    invitation.router,
```

Comment:
By mistake?

Reply:
No :)
As the description says, the `notification` route contains in it the `invitation` route